app/pages/time/index.page.tsx

```diff
@@ -102,9 +139,31 @@ const Time = (): JSX.Element => {
           <BiPlus className="flex items-end text-3xl" />
         </Button>
       </div>
+      
+      <span className="text-center text-xs">
+      <Link href={`/time?date=${previousWeekString}`}>
+        <a
+          className={`k w-10 px-2 rounded-l-lg py-1 bg-gray-400`}
+        >
+        
+       Last
+
+        </a>
+      </Link>
+      <Link href={`/time?date=${nextWeekString}`}>
+        <a
+          className={` w-10 px-2 rounded-r-lg py-1 bg-gray-400`}
+        >
+          Next
+        </a>
+      </Link>
+    </span>
+      <div className='text-lg font-semibold'>This Week: {startDate + "-" + endDate}</div>
```

Comment:
I would prefer `This Week: {startDate} - {endDate}`, but that's just a personal taste.